Makefile

```diff
@@ -7,6 +7,10 @@ PEM := $(PEM_DIR)/cert.pem
 NAMESERVER := 8.8.8.8
 RESOLV_CONF_DIR := endpoint/endpointComp
 OUTPUT_BASE_DIR := output_base
+# Endpoint build target. The default intends to the platform of your development system.
+TARGET := development_host
+ENABLE_ENDPOINT_TEST := false
```

Comment:
If `TEST` is enabled, it implies all of available test programs will be built and ready for use. We don't really need the option `ENABLE_ENDPOINT_TEST`.